### ISSUE TITLE
Selecting and changing roles 

### ISSUE THREAD

**Issue by csarven** [SolidOS/solid-panes#222]
re https://github.com/solid/userguide/#role

>Tip: You need to refresh your web browser for the role changes to take affect.

Why should the user do this? Is user's commitment of the selection required? Is the required commitment indicated in the UI eg. form submit botton? If commitment action is not part of the UI, why can't on selection trigger the changes?

**Comment by michielbdejong** [comment]
Good point! Transferring this to the solid-panes repo, which is where we could fix this.
See https://github.com/solid/solid-panes/blob/master/src/dashboard/preferencesFormText.ttl
